docker/package/wizard_structure.py

```diff
@@ -18,6 +18,7 @@
 protocol_hash_regex = (
     b"P[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{50}"
 )
+signer_uri_regex = b"((?:tcp|unix|https|http):\/\/.+)\/keys\/(tz[123]\w{33})\/?"
```

Comment:
Does `keys` part of the URL actually needed?
```suggestion
signer_uri_regex = b"((?:tcp|unix|https|http):\/\/.+)\/(tz[123]\w{33})\/?"
```